templates/ObjectSchema.stencil

```diff
@@ -0,0 +1,20 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+
+import AzureCore
+import Foundation
```

Comment:
Perhaps standard imports should be in a small stencil snippet as well?

Reply:
Move to IncludeSnippet.stencil